controls/radrichtextbox/features/commands.md

```diff
@@ -148,3 +190,170 @@ If you want to handle application commands on RadRichTextBox owner's level, you
 {{endregion}}
 
 {% endif %}
+
+## Command-related Events
+
+**RadRichTextBox** exposes two events that allow you interfere the process of executing a command.

```

Comment:
I am not sure but I think here it should be "two events that allow**s**"

Reply:
the events allow you/they allow you - it is in plural, so it should be without the 's'